app/src/org/commcare/models/FormRecordProcessor.java

```diff
@@ -85,7 +85,8 @@ public TransactionParser getParser(KXmlParser parser) {
 
         //Let anyone who is listening know!
         Intent i = new Intent("org.commcare.dalvik.api.action.data.update");
-        this.c.sendBroadcast(i);
+        i.putStringArrayListExtra("cases", factory.getCreatedAndUpdatedCases());
```

Comment:
Ah, good call. I've added the case read permission and an example of it being read in https://github.com/dimagi/commcare-tester-app/pull/4